metadata/index.html

```diff
@@ -737,7 +735,7 @@
             <dt id="table-notes"><code>notes</code></dt>
             <dd>
               <p>
-                An <a>array property</a> that provides an array of objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
+                A <a>common property</a> that provides one or more objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
```

Comment:
It feels a bit weird to call this a common property, as common properties can go on any type of description object (table group, table, column etc etc). I think this should stay being an array property.